### PR TITLE
[10.0][FIX] import queue job failing

### PR DESCRIPTION
**Pull request by sergiocorato**
v. 8 similar bug https://github.com/OCA/account-financial-tools/issues/280 
(not sure for `Job` [definition](https://github.com/OCA/account-financial-tools/pull/802/files#diff-600ba28ee31d3c85639690a7b43faaa9R19), but it works)

### CODE REVIEW

**Review by Tonow-c2c — APPROVED** [review]
LGTM thanks

File: account_move_batch_validate/models/account_move.py [hunk 1]
```diff
@@ -2,13 +2,22 @@
...
+try:
+    from odoo.addons.queue_job.job import job, Job
+except ImportError:
+    _logger.debug('Can not `import queue_job`.')
+    import functools
+
+    def empty_decorator_factory(*argv, **kwargs):
+        return functools.partial
+    job = empty_decorator_factory
+    Job = empty_decorator_factory
```
Comment (guewen) [inline account_move_batch_validate/models/account_move.py]: `Job = empty_decorator_factory` is not required as it is only called in a method

File: account_move_batch_validate/wizard/account_move_validate.py [hunk 2]
```diff
@@ -3,7 +3,20 @@
 # License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
 
 from odoo import api, fields, models
-from odoo.addons.queue_job.job import job
+import logging
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from odoo.addons.queue_job.job import job, Job
```
Comment (guewen) [inline account_move_batch_validate/wizard/account_move_validate.py]:
```suggestion
    from odoo.addons.queue_job.job import job
```

File: account_move_batch_validate/wizard/account_move_validate.py [hunk 3]
```diff
@@ -3,7 +3,20 @@
...
+except ImportError:
+    _logger.debug('Can not `import queue_job`.')
+    import functools
+
+    def empty_decorator_factory(*argv, **kwargs):
+        return functools.partial
+    job = empty_decorator_factory
+    Job = empty_decorator_factory
```
Comment (guewen) [inline account_move_batch_validate/wizard/account_move_validate.py]:
```suggestion
```

**Review by guewen — CHANGES_REQUESTED** [review]

File: account_move_batch_validate/models/account_move.py (hunk 1 above)
Comment (sergiocorato) [reply account_move_batch_validate/models/account_move.py]: thanks

**Review by guewen — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Comment by alexis-via** [comment]
flake8 says:
```
account_move_batch_validate/models/account_move.py:80:23: F821 undefined name 'Job'
```
You removed the import of Job, but it's still used in the code (line 80 of account_move.py)

File: account_move_batch_validate/wizard/account_move_validate.py (hunk 2 above)
Comment (sergiocorato) [reply account_move_batch_validate/wizard/account_move_validate.py]: > 
flake8 says:

account_move_batch_validate/models/account_move.py:80:23: F821 undefined name 'Job'

You removed the import of Job, but it's still used in the code (line 80 of account_move.py)

@guewen @alexis-via so better mantain this `Job` import?

Comment (sergiocorato) [reply account_move_batch_validate/wizard/account_move_validate.py]: > > 
> 
> flake8 says:
> 
> account_move_batch_validate/models/account_move.py:80:23: F821 undefined name 'Job'
> 
> You removed the import of Job, but it's still used in the code (line 80 of account_move.py)
> 
> @guewen @alexis-via so better mantain this `Job` import?

@sebalix too

**Pull request closed without merging**